vms/vms/views.py

```diff
@@ -0,0 +1,18 @@
+from django.shortcuts import redirect
+from django.views.generic.edit import FormView
+
+from django.core.mail import send_mail
+from django.core.mail.message import BadHeaderError
+from django.http.response import HttpResponse
+
+from vms import settings
+
+def anonymous_required(func):
+    def as_view(request, *args, **kwargs):
```

Comment:
@tapasweni-pathak The mixin that we used is AnonymousRequiredMixin. So thought of giving this name for better understanding